src/__tests__/authentication.test.ts

```diff
@@ -16,6 +20,7 @@ describe("authenticateRequest", () => {
   it("should successfully verify authenticity", async () => {
     const token = jwt.sign({ foo: "bar" }, MOCK_SIGNING_KEY);
     await authenticateRequest(token, "tenant");
+    expect(true).toBe(true);
```

Comment:
Yeah I think the reason for no return value was because it avoided that annoying return some value and then have to check it and throw an error kind of behaviour
I've added in the above now